### PR TITLE
ENH: Change DIPY repository organization to `dipy`

### PR DESCRIPTION
**Pull request by jhlegarreta**
Change DIPY repository organization to `dipy`.

### CODE REVIEW

**Comment by vsoch** [comment]
hang tight @jhlegarreta I'll see if I can fix the CI - it's run so infrequently the ruby version is ancient.

**Comment by vsoch** [comment]
hey @jhlegarreta could you please rebase with master to get the latest change - I just updated the Gemfile version of ruby so the build will run successfully and we can preview (and test) your change. Thank you!

**Comment by jhlegarreta** [comment]
> hey @jhlegarreta could you please rebase with master to get the latest change - I just updated the Gemfile version of ruby so the build will run successfully and we can preview (and test) your change. Thank you!

 Push-forced 79547d8 after rebasing on `master`.

**Comment by vsoch** [comment]
okay let's take a look - https://138-1283579-gh.circle-artifacts.com/0/nipy.github.com/index.html

File: help.html [hunk 1]
```diff
@@ -22,4 +22,4 @@ <h2>Neurostars</h2>
 
 <h2>Github issues</h2>
 
-<p>Most of the projects have an issues page on which you are (very) welcome to report about problems that you have encountered using the software. For example, for issues related to the Dipy software library, please go here: <a href="https://github.com/nipy/dipy/issues" target="_blank">https://github.com/nipy/dipy/issues</a>. More informal conversations can be struck on the gitter pages of the projects (e.g. <a href="https://gitter.im/nipy/dipy" target="_blank">https://gitter.im/nipy/dipy)</a></p>.
+<p>Most of the projects have an issues page on which you are (very) welcome to report about problems that you have encountered using the software. For example, for issues related to the Dipy software library, please go here: <a href="https://github.com/dipy/dipy/issues" target="_blank">https://github.com/dipy/dipy/issues</a>. More informal conversations can be struck on the gitter pages of the projects (e.g. <a href="https://gitter.im/nipy/dipy" target="_blank">https://gitter.im/nipy/dipy)</a></p>.
```
Comment (vsoch) [inline help.html]: Is there still a gitter channel at nipy/dipy (see last line of this section) or does that need to be updated too?

Comment (vsoch) [reply help.html]: Actually I just checked and confirmed that there is - so this is correct! This LGTM, let's merge.

Comment (jhlegarreta) [reply help.html]: That's right, the DIPY gitter channel still dwells under the NIPY umbrella.

Thanks for having taken care of the PR.

Comment (vsoch) [reply help.html]: Of course! Thanks for fixing the links.

**Pull request merged**